src/phone-input/phone-input-alternative.js

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React, {useRef} from 'react';
+import {COUNTRIES} from './constants.js';
+import CountryPicker from './country-picker.js';
+import defaultProps from './default-props.js';
+import {StyledPhoneInputRoot, StyledDialCode} from './styled-components.js';
+import {Input as DefaultInput} from '../input/index.js';
+import {getOverrides, mergeOverrides} from '../helpers/overrides.js';
+import type {LitePropsT} from './types.js';
+
+PhoneInputAlternative.defaultProps = {
+  ...defaultProps,
+  countries: COUNTRIES,
+  clearable: true,
+};
+
+export default function PhoneInputAlternative(props: LitePropsT) {
```

Comment:
Could the name be more descriptive of the result? Does design what this to eventually be the default? If so, maybe add `Next` suffix and add a warning for default usage.

Reply:
I named it with the intent of coexisting components. And since the API is the same at this point it could potentially be silently replaced. Unless we want to remove the usage of the non-split version completely before the v10 I think the warning might be a little too noisy. Happy to change the naming.

Reply:
If it's safe to replace with the new version? I assume the two have different overrides. My main concern is that `Alternate` does not communicate how it's different from the default. Maybe `Separated`?

Reply:
There is a wrapper root element added for the split version to contain the two separate controls.